distribution/lib/Standard/Table/0.0.0-dev/src/Value_Type.enso

```diff
@@ -304,8 +304,8 @@ type Value_Type
             Value_Type.Float _ -> True
             Value_Type.Decimal _ _ -> True
             _ -> False
-        Value_Type.Float self_bits -> case target_type of
-            Value_Type.Float target_bits -> target_bits >= self_bits
+        Value_Type.Float _ -> case target_type of
+            Value_Type.Float _ -> True
```

Comment:
Shall we add a comment that this is valid until we get `f32` support on in-memory? I assume once that happens, we probably want to revert this change.

We can even link a ticket: https://github.com/enso-org/enso/issues/6109

Reply:
I don't know if we want to revert it. An automatic cast with a warning seems more useful than an error telling me I have to do a manual cast. I'll add a comment to think about it.

Reply:
Hmm, I think you are right.

I'm not a fan of this because this is a **write** operation, that cannot be easily undone. So if the user sees the warning it is already too late to address it, all they can do is acknowledge it.

In this case though, in most cases the user would indeed cast as there isn't much they can do. And float is an inexact type anyway - so I guess the (additional) loss of precision is not that big of a deal here as it would be elsewhere.

So happy to keep it indeed.

Reply:
But with dry run mode the user gets to see the warning before they make the write.

Reply:
Fair enough! I forgot about that case.